app/controllers/reviews_controller.rb

```diff
@@ -24,25 +24,26 @@ def create
   end
 
   def edit
-    @sitcom = Sitcom.find(params[:sitcom_id])
-    @review = @sitcom.reviews.find(params[:id])
+    @review = Review.find(params[:id])
+    @sitcom = @review.sitcom
   end
 
   def update
-    @sitcom = Sitcom.find(params[:sitcom_id])
-    @review = @sitcom.reviews.find(params[:id])
+    @review = Review.find(params[:id])
+    @sitcom = @review.sitcom
```

Comment:
This is a good example where one of the possible returns for this controller is to `render` a template, so setting the instance variables in this controller is good because they are used in the view that it can render.